R/bs4_book.R

```diff
@@ -615,3 +664,43 @@ bs4_check_dots <- function(...) {
     )
   }
 }
+
+# test helpers -----------------------------------------------------------------
```

Comment:
it's plural but there's only one for now :-)

Reply:
is this only for test ? Meaning test helpers are stored inside the source of package ? 
We usually store theme inside the test folder directly. 

Also this function duplicates with `bookdown::book_skeleton()`, or the internal `bookdown_skeleton()` used for template. Maybe we can adapt those for the need of `bs4_book()`. It should be just a matter of rendering with special format (`render_book(output_format = bookdown::bs4_book())`)

Reply:
So the function should actually

* call `bookdown::book_skeleton`
* modify files if needed (content, YAML configs)
* call `render_book`?

Reply:
> modify files if needed (content, YAML configs)

Do you need to have special content in the Rmd for your test ? You should be able to control most of the config behavior from `render_book()`